src/test/java/teammates/test/cases/storage/StudentsDbTest.java

```diff
@@ -63,8 +54,6 @@ public void testDefaultTimestamp() throws InvalidParametersException, EntityAlre
         ______TS("success : defaultTimeStamp for updatedAt date");
         
         assertEquals(defaultStudentCreationTimeStamp, student.getUpdatedAt());
-        
-        studentsDb.deleteStudent(s.course, s.email);
```

Comment:
It <strong>is</strong> the purpose of this PR. Previously, after each tests, the corresponding entities are kept. We now want them to be retained, hence this is deleted.